inc/wpseo-functions.php

```diff
@@ -9,6 +9,20 @@
 	exit();
 }
 
+if ( ! function_exists( 'wp_rest_api_available' ) ) {
+	/**
+	 * Checks if the WP-REST-API with at least version 2.0 is available.
+	 *
+	 * @return bool
+	 */
+	function wp_rest_api_available() {
+		if ( defined( 'REST_API_VERSION' ) && version_compare( REST_API_VERSION, '2.0', '>=' ) ) {
```

Comment:
`defined( 'REST_API_VERSION' ) && version_compare( REST_API_VERSION, '2.0', '>=' ) ` is returning a boolean. I would return this immediately instead of doing the `if`